src/app.json.template

```diff
@@ -10,12 +10,8 @@
         "odt"
     ],
     "companyName": "[ COMPANY NAME HERE ]",
-    "defaultLocale": "en-US",
-    "supportedLocales": [
-        "en-US",
-        "fr-FR"
-    ],
-    "minUploadSize": 30720,
+    "defaultLocale": "en-EU",
+    "minUploadSize": 3720,
```

Comment:
Min upload size was 30kb, some candidates could have a PDF that is less that. Lowered to 3KB.